src/backend/core/api/viewsets.py

```diff
@@ -191,6 +198,142 @@ def get_me(self, request):
         )
 
 
+class StatsViewSet(viewsets.GenericViewSet):
+    """
+    API endpoint that provides various statistics in JSON format.
+    """
+
+    permission_classes = [IsAdminUser]
+
+    def list(self, request):
```

Comment:
This is not a viewset which is needed here

```suggestion
class StatsAPIView(views.APIView):
    """
    API endpoint that provides various statistics in JSON format.
    """

    permission_classes = [IsAdminUser]

    def get(self, request, *args, **kwargs):
```